### PR TITLE
[Lib Support] hot model update

### LINKED ISSUE #104: Incremental learning supports hot model updates.
**What would you like to be added/modified**:
Currently, models are updated through restart. infer worker.  Incremental learning is required to support hot model updates rather than only through restart.

**Why is this needed**:
inference engine supports model reloading.  Therefore, it is hoped that dynamic update can be supported.

### PR DESCRIPTION
**Pull request by JoeyHwong-gk**
fix #104 #7 
Signed-off-by: JoeyHwong <joeyhwong@gknow.cn>

### CODE REVIEW

**Comment by JimmyYang20** [comment]
suggest to modify the PR title to `Lib supports hot model updates`

File: lib/sedna/core/base.py [hunk 1]
```diff
@@ -12,21 +12,114 @@
...
+        self.job_kind = job_kind
+        self.namespace = namespace
+        self.worker_name = worker_name
+        self.lc_server = lc_server
+        self.version = version
+        temp_path = self.production_estimator.model_save_path or "/tmp"
+        self.temp_path = os.path.dirname(temp_path)
+        super(ModelLoadingThread, self).__init__()
+
+    def report_task_info(self, status, kind="deploy"):
```
Comment (ugvddm) [inline lib/sedna/core/base.py]: why is it not a common method? or use a common method? It's not necessary!

File: lib/sedna/core/base.py [hunk 2]
```diff
@@ -12,21 +12,114 @@
...
+                    continue
+            if not (model and FileOps.exists(model)):
+                continue
+            if latest_version == self.version:
+                continue
+            self.version = latest_version
+            status = K8sResourceKindStatus.RUNNING.value
+            self.report_task_info(status=status)
+            with self.MODEL_MANIPULATION_SEM:
+                LOGGER.info(f"Update model start with version {self.version}")
```
Comment (ugvddm) [inline lib/sedna/core/base.py:91]: it not necessary to print a info log for every `load model`

File: lib/sedna/core/base.py (hunk 1 above)
Comment (JoeyHwong-gk) [reply lib/sedna/core/base.py]: Yep, It's a common method. you can review the latest version

**Pull request merged**